server/logout.php

```diff
@@ -0,0 +1,7 @@
+<?php
+	$root = realpath($_SERVER["DOCUMENT_ROOT"]);
```

Comment:
Do you think this functionality would be better put in server/callbacks.php? That's where all the login/register/password reset stuff is, so I think it makes sense to keep all this functionality together in that file.

Reply:
on a sidenote, we should also rename server/callbacks.php I think...

Reply:
Agreed on both.